Plugins/FastJet/include/Acts/Plugins/FastJet/TrackJets.hpp

```diff
@@ -0,0 +1,71 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <Acts/Definitions/Units.hpp>
+
+#include <optional>
+#include <vector>
+
+#include <fastjet/JetDefinition.hh>
+#include <fastjet/PseudoJet.hh>
```

Comment:
This is not critical, but I'm wondering if there's a way to push these a `.cpp` file, and use forward declarations here. If I'm reading the code right, in the end fastjet only receives four-vectors, so in principle the class could be split into a (non-templated) part that deals with input four-vectors and stores the cluster sequence, and another part that creates the inputs from the `TrackContainer::TrackProxy` objects and stores the track container.

The implementation of the former could then move to a `.cpp` file.

Reply:
I've attempted this with commit 2fa98744f0 -- let me know if this is what you had in mind @paulgessinger